huaweicloud/services/rds/resource_huaweicloud_rds_pg_table_restore.go

```diff
@@ -0,0 +1,254 @@
+package rds
+
+import (
+	"context"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+var pgTableRestoreNonUpdatableParams = []string{"instance_id"}
+
+// @API RDS POST /v3/{project_id}/instances/batch/restore/tables
+func ResourceRdsPgTableRestore() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourcePgTableRestoreCreate,
+		ReadContext:   resourcePgTableRestoreRead,
+		DeleteContext: resourcePgTableRestoreDelete,
+
+		CustomizeDiff: config.FlexibleForceNew(pgTableRestoreNonUpdatableParams),
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"instance_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"restore_time": {
+				Type:     schema.TypeInt,
+				Required: true,
+				ForceNew: true,
+			},
+			"databases": {
+				Type:     schema.TypeList,
+				Elem:     pgTableRestoreDatabaseSchema(),
+				Required: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func pgTableRestoreDatabaseSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"database": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"schemas": {
+				Type:     schema.TypeList,
```

Comment:
```suggestion
				Type:     schema.TypeSet,
```
if the order of the schemas has been changed, the data should not be updated, if the type is list, then it will be updated, if the type is set, then it will not, and fix other places